locales/en.yml

```diff
@@ -3,3 +3,38 @@ en:
     example:
       name: 'Citizens Advice'
       body: 'This is some body text'
+    breadcrumbs:
+      home: Home
+    callouts:
+      example: Example
+      important: Important
+      adviser: Adviser
+    form:
+      optional: optional
+    logo:
+      title: Citizens Advice homepage
+    oisc_warning:
+      close: Close this warning
+    page_review:
+      body_html: Page last reviewed on <span class="cads-page-review-date">%{date}</span>
```

Comment:
Could we avoid having HTML in the translations? I've also just updated this markup as part of https://github.com/citizensadvice/design-system/pull/392 which would conflict with this. This could probably be:

```
page_review
  last_reviewed: Page last reviewed on
```

Reply:
It's a good idea to have HTML in the translation so that the full context of the translation is taken into account by the translator. 

Are you sure that the correct localisation in welsh would put the date as the last item in the sentence?

Reply:
Ah interesting. Fairly sure in the case of Welsh the construction is the same, but I take your point. I still think it makes things brittle, but I reckon the part I object to most is having _class names_ in here. Looking at the component all `cads-page-review-date` does is make the text bold, so perhaps this could be:

page_review:
      body_html: Page last reviewed on <strong>%{date}</strong>

Reply:
Sure that's not a problem.